nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors/src/test/java/org/apache/nifi/processors/splunk/TestQuerySplunkIndexingStatus.java

```diff
@@ -121,16 +121,44 @@ public void testMoreIncomingFlowFileThanQueryLimit() throws Exception {
     }
 
     @Test
-    public void testTimedOutEvents() throws Exception {
+    public void testAckCheckedIsTrueAndFlowFileWithTimedOutEvents() throws Exception {
         // when
-        testRunner.enqueue(givenFlowFile(1, System.currentTimeMillis() - TimeUnit.HOURS.toMillis(2)));
+        testRunner.enqueue(givenFlowFile(1, System.currentTimeMillis() - TimeUnit.HOURS.toMillis(2), true));
         testRunner.run();
 
         // then
         Mockito.verify(service, Mockito.never()).send(Mockito.anyString(), Mockito.any(RequestMessage.class));
         testRunner.assertAllFlowFilesTransferred(QuerySplunkIndexingStatus.RELATIONSHIP_UNACKNOWLEDGED, 1);
     }
 
+    @Test
+    public void testAckCheckedIsFalseAndTimedOutEventFlowFileWithAcknowledgeResponse() throws Exception {
+        // when
+        final Map<Integer, Boolean> acks = new HashMap<>();
```

Comment:
Minor: you can make it more readable and shorter with Collections.singletonMap
Also:  this should be part of the given block

Reply:
added to singletonMap